sdk/identity/azure-identity/azure/identity/_constants.py

```diff
@@ -10,6 +10,9 @@
 DEFAULT_REFRESH_OFFSET = 300
 DEFAULT_TOKEN_REFRESH_RETRY_DELAY = 30
 
+CACHE_NON_CAE_SUFFIX = ".nocae"  # cspell:disable-line
```

Comment:
If we add suffix for nocase, is this a breaking change?

Reply:
e.g. if one app uses old version of identity and the other uses a new one. Even neither of them enables CAE, they cannot share tokens.

Reply:
I can add a note about this in the CHANGELOG. No other languages seems to have mentioned this in their changelogs. I suppose the impact wouldn't be that great, as it would just be an additional token request, and an additional file in the cache directory.

Reply:
The problem is the token persisted by app1 will not be shared to app2.

Some apps freeze the library versions.

Seems to me it will break some scenarios.

Reply:
I did note this type of scenario in the CHANGELOG under `Breaking Changes`, and I could probably add a TROUBLESHOOTING section for this as a followup. Might be tricky to start trying to fallback to caches with names without the suffixes added on. and I'm not certain if it'd even be worth the additional complexity.